docs/submit/osg-flock.md

```diff
@@ -1,97 +1,170 @@
 
+1. Send email to <help@opensciencegrid.org> requesting the pool password file for flocking.
 
-```file
-   #-- Flock to the OSG using pool password
-   SEC_PASSWORD_FILE = /etc/condor/pool_password
-   SEC_DEFAULT_AUTHENTICATION_METHODS = FS,PASSWORD
-```    
+1. Place the pool password file in `/etc/condor/pool_password`.
 
+1. Create a configration file named `/etc/condor/config.d/90-flock-pool-password.conf` with the following contents:
 
+        :::ini
+        #-- Flock to OSG using pool password
+        SEC_PASSWORD_FILE = /etc/condor/pool_password
+        SEC_DEFAULT_AUTHENTICATION_METHODS = FS,PASSWORD
 
-Project Names
--------------
 
-OSG will only run jobs tagged with a valid *ProjectName* - this is the main attribute
-used for accounting. Please open at ticket to register a new project.
-Jobs should specify which project to be accounted against by adding
-the *+ProjectName* attribute. Note that the value is a string and hence
-the double quotes are required. For example:
+Managing Services
+-----------------
+You will need the following services enabled and running:
+* gratia-probes-cron
+* condor
+
+The following table gives the commands needed to start, stop, enable, and disable a service:
+
+To...                                       | On EL6, run the command...              | On EL7, run the command...         |
+:------------------------------------------ | :-------------------------------------- | :--------------------------------- |
+| Start a service                           | `service <SERVICE-NAME> start`          | `systemctl start <SERVICE-NAME>`   |
+| Stop a service                            | `service <SERVICE-NAME> stop`           | `systemctl stop <SERVICE-NAME>`    |
+| Enable a service to start on boot         | `chkconfig <SERVICE-NAME> on`           | `systemctl enable <SERVICE-NAME>`  |
+| Disable a service from starting on boot   | `chkconfig <SERVICE-NAME> off`          | `systemctl disable <SERVICE-NAME>` |
+
+
+Usage
+-----
+### Running jobs in OSG
+Users should be aware that OSG jobs are distributed across multiple institutions across a large geographical area.
+Each institution will have its own policy about the kinds of jobs that are allowed to run,
+and data transfer may be more complicated.
+The [OSG Helpdesk Solutions](https://support.opensciencegrid.org/support/solutions) page has information about
+what users should know; the "Choosing Resources for Jobs" and "Data Management" sections should be particularly
+relevant.
+
+### Specifying a project
+OSG will only run jobs that have a registered *project* associated with them.
+A project is associated with a job by adding a ProjectName line to the submit file.
+For example:
 
 ```file
-+ProjectName = "Some_Name_Here"
++ProjectName = "My_Project"
 ```
+The double quotes are necessary.
 
-Get Help
----------------
+In addition, the project must be registered in OSG Topology.
```

Comment:
Isn’t this really the first step? I.e., first you register the project in Topology, then you are able to change your submit file and flock.